pkg-manifest/read-project-manifest/src/index.ts

```diff
@@ -14,12 +14,21 @@ import {
   readJsonFile,
 } from './readFile'
 
-type WriteProjectManifest = (manifest: ProjectManifest, force?: boolean) => Promise<void>
+interface WriteProjectManifestOptions {
+  force?: boolean
+  manifestPathOverride?: string
```

Comment:
The existing behavior was to re-write the file in same location it was loaded from. That is unwanted for our use case, so provided this override option.

This "breaks" existing options syntax, although I did not find any usage of the force option... so actually worked in-repo.

If this method is considered part of pnpm's public API though, I can tweak to accept a boolean _or_ this options object for backwards compatibility.

Alternatively, I can use the `writeProjectManifest` function directly in `directory-fetcher`, but I did appreciate that the `manifest.writeProjectManifest` came with sensible defaults

Reply:
Don't add this option. Just use a function that accepts the path to the manifest for saving the manifest.